### PR TITLE
Add Pago Facil, RapiPago, Baloto specs

### PR DESCRIPTION
**Pull request by mikhail-kuzmiankou-cko**
These payment methods are already available in QA. 
Preview: https://api-reference.checkout.com/preview/task/da-126-add-pagofacil-rapipago-baloto-specs/

### CODE REVIEW

File: spec/components/schemas/Payments/ResponseSources/PaymentResponseRapiPagoSource.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            type: string
+            description: 'The type of the integration. A redirect url is returned, which the merchant redirects the customer to, to continue payment.'
+          description:
+            maxLength: 200
+            type: string
+            description: 'Payment description'
+      -
+        type: object
+        properties:
+          ticket:
```
Comment (kevin-reece-cko) [inline spec/components/schemas/Payments/ResponseSources/PaymentResponseRapiPagoSource.yaml]: RapiPago doesn't support DIRECT tickets

Comment (mikhail-kuzmiankou-cko) [reply spec/components/schemas/Payments/ResponseSources/PaymentResponseRapiPagoSource.yaml]: Good catch, fixed!

File: spec/components/schemas/Payments/RequestSources/PaymentRequestBalotoSource.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  -
+    type: object
+    required:
+      - integration_type
+      - country
+      - payer
+    properties:
+      integration_type:
+        enum:
+          - redirect
```
Comment (philippe-leonhardt-cko) [inline spec/components/schemas/Payments/RequestSources/PaymentRequestBalotoSource.yaml]: Does baloto only provide the redirect integration type?

File: spec/components/schemas/Payments/RequestSources/PaymentRequestRapiPagoSource.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  -
+    type: object
+    required:
+      - integration_type
+      - country
+      - payer
+    properties:
+      integration_type:
+        enum:
+          - redirect
```
Comment (philippe-leonhardt-cko) [inline spec/components/schemas/Payments/RequestSources/PaymentRequestRapiPagoSource.yaml]: Does RapiPago only offer the redirect integration type?

File: spec/components/schemas/Payments/RequestSources/PaymentRequestBalotoSource.yaml (hunk 2 above)
Comment (mikhail-kuzmiankou-cko) [reply spec/components/schemas/Payments/RequestSources/PaymentRequestBalotoSource.yaml]: Yes, only redirect type.

File: spec/components/schemas/Payments/RequestSources/PaymentRequestRapiPagoSource.yaml (hunk 3 above)
Comment (mikhail-kuzmiankou-cko) [reply spec/components/schemas/Payments/RequestSources/PaymentRequestRapiPagoSource.yaml]: Yes, according to dLocal, only Pago Facil supports direct type in Argentina.

**Pull request merged**